Traceback (most recent call last):
  File "C:\Users\segun\Desktop\Projects\Meme_bot\omo_bot.py", line 5, in <module>
    from dotenv import load_dotenv
ModuleNotFoundError: No module named 'dotenv'
